examples/research_projects/consistency_training/train_cm_ct_unconditional.py

```diff
@@ -1255,8 +1255,8 @@ def unwrap_model(model):
             c_in_teacher = get_input_preconditioning(teacher_timesteps, input_precond_type=args.input_precond_type)
             c_in_student = get_input_preconditioning(student_timesteps, input_precond_type=args.input_precond_type)
 
-            c_skip_teacher, c_out_teacher = scalings_for_boundary_conditions(teacher_timesteps)
-            c_skip_student, c_out_student = scalings_for_boundary_conditions(student_timesteps)
+            c_skip_teacher, c_out_teacher = scalings_for_boundary_conditions(teacher_timesteps, args.sigma_min)
+            c_skip_student, c_out_student = scalings_for_boundary_conditions(student_timesteps, args.sigma_min)
```

Comment:
Are we defaulting to reasonable values for these args?

Reply:
`sigma_min` defaults to `0.002`, which is the value of $`\sigma_{min}`$/$`\epsilon`$ used in the [original Consistency Models paper](https://arxiv.org/pdf/2303.01469.pdf) and the [improved Consistency Training paper](https://arxiv.org/pdf/2310.14189.pdf) (see Table 1 in the latter paper), which I believe is a reasonable default value.

(Note that `sigma_min` should be a small positive value rather than `0` to avoid numerical issues when using ODE solvers.)

Reply:
Cool! Do you think it makes sense to add this info to the README as well?

Reply:
I have added documentation on the consistency training-specific hyperparameters to the README.